modules/ROOT/pages/hdfs/hdfs-connector-reference.adoc

```diff
@@ -1,649 +1,603 @@
+| Payload a| Binary |  The payload to append to the file. |  #[payload] | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
 |===
 
 
 === For Configurations
+<<hdfs>> 
 
-* <<hdfs>>
-
-=== Throws
+==== Throws
 
-* HDFS:CONNECTIVITY
-* HDFS:INVALID_REQUEST_DATA
-* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA
-* HDFS:RETRY_EXHAUSTED
-* HDFS:UNKNOWN
+* HDFS:CONNECTIVITY 
+* HDFS:INVALID_REQUEST_DATA 
+* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA 
+* HDFS:RETRY_EXHAUSTED 
+* HDFS:UNKNOWN 
 
 
 [[copyFromLocalFile]]
-== Copy From Local File
-
+=== Copy From Local File
 `<hdfs:copy-from-local-file>`
 
 
-Copy the source file on the local disk to the FileSystem at the given target path, set deleteSource if the source should be removed.
-
+Copy the source file on the local disk to the file system for a target path, set *Delete Source* if the source should be removed.
 
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Delete Source a| Boolean |  whether to delete the source. |  false |
-| Overwrite a| Boolean |  whether to overwrite destination content. |  true |
-| Source a| String |  the source path on the File System. |  | x
-| Destination a| String |  the target path on the local disk. |  | x
+| Delete Source a| Boolean |  Whether to delete the source. |  false | 
+| Overwrite a| Boolean |  Whether to overwrite destination content. |  true | 
+| Source a| String |  The source path in the file system. |  | x
+| Destination a| String |  The target path on the local disk. |  | x
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
 |===
 
 
 === For Configurations
+<<hdfs>> 
 
-* <<hdfs>>
-
-=== Throws
-
-* HDFS:CONNECTIVITY
-* HDFS:INVALID_REQUEST_DATA
-* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA
+==== Throws
+* HDFS:CONNECTIVITY 
+* HDFS:INVALID_REQUEST_DATA 
+* HDFS:INVALID_STRUCTURE_FOR_INPUT_DATA 
 * HDFS:RETRY_EXHAUSTED
-* HDFS:UNKNOWN
+* HDFS:UNKNOWN  
 
 
 [[copyToLocalFile]]
-== Copy To Local File
-
+=== Copy To Local File
 `<hdfs:copy-to-local-file>`
 
 
-Copy the source file on the FileSystem to local disk at the given target path, set deleteSource if the source should be removed. useRawLocalFileSystem indicates whether to use RawLocalFileSystem as it is a non CRC File System.
-
+Copy the source file in the file sSystem to a local disk at the given target path, set *Delete Source* if the source should be removed. *Use Raw Local File System* indicates whether to use RawLocalFileSystem as it is a non-CRC file system.
```

Comment:
Fix run-on. Typo "sSystem". What does it mean to say "as it is a non-CRC file system"?